javalib/src/main/scala/java/util/Formatter.scala

```diff
@@ -114,6 +114,25 @@ final class Formatter(private val dest: Appendable) extends Closeable with Flush
             result
           }
 
+          def formatThousands(value: String, decimalSep: Char = '.', thousandsGrouping: String = ","): String = {
+            val parts = value.split(decimalSep).toList
+            val integer = parts.head
+            val decimal = parts.tail
```

Comment:
Use: `val integer :: restParts = value.split(decimalSep, -1).toList` (-1 is just for safety, in case we have "1.")